lib/v2/datasource/remote/api/seeds_history_repository.dart

```diff
@@ -0,0 +1,30 @@
+import 'package:async/async.dart';
+import 'package:http/http.dart' as http;
+import 'package:seeds/v2/datasource/remote/api/network_repository.dart';
+import 'package:seeds/v2/datasource/remote/model/seeds_history_model.dart';
+
+class SeedsHistoryRepository extends NetworkRepository {
+
+  Future<Result<dynamic>> getNumberOfTransactions(String userAccount) {
+    print('[http] get seeds seeds history for account: $userAccount ');
+
+    final String request = createRequest(
```

Comment:
Look at how other requests are done. 
You need to come the "histry.seeds" and other strings into the constants in the base class.

Reply:
Done 👍